api/lib/infrastructure/repositories/jury-certification-repository.js

```diff
@@ -84,15 +90,28 @@ async function _toDomainWithComplementaryCertifications({ juryCertificationDTO,
       })
   );
 
-  const cleaCertificationResult = await cleaCertificationResultRepository.get({
-    certificationCourseId: juryCertificationDTO.certificationCourseId,
-  });
-  const pixPlusDroitMaitreCertificationResult = await pixPlusDroitMaitreCertificationResultRepository.get({
-    certificationCourseId: juryCertificationDTO.certificationCourseId,
-  });
-  const pixPlusDroitExpertCertificationResult = await pixPlusDroitExpertCertificationResultRepository.get({
-    certificationCourseId: juryCertificationDTO.certificationCourseId,
-  });
+  const partnerCertifications = _.compact(juryCertificationDTO.partnerCertifications).map(PartnerCertification.from);
+
+  const cleaPartnerCertification = partnerCertifications.find(({ partnerKey }) =>
+    [PIX_EMPLOI_CLEA, PIX_EMPLOI_CLEA_V2].includes(partnerKey)
+  );
+  const cleaCertificationResult = cleaPartnerCertification
+    ? CleaCertificationResult.buildFrom(cleaPartnerCertification)
+    : CleaCertificationResult.buildNotTaken();
```

Comment:
On le fait à pas mal d'endroit, est ce qu'on ne devrait pas faire un buildFrom qui renvoie buildNotTaken si la valeur est vide
```suggestion
  const cleaCertificationResult = CleaCertificationResult.buildFrom(cleaPartnerCertification);
```

Reply:
Commit temporaire, la version finale n'utilise plus CleaCertificationResult et les équivalents Pix+ Droit